### ISSUE TITLE
Add tab shortcuts to Scene Docker

### ISSUE THREAD

**Issue by ShatReal** [godotengine/godot-proposals#2725]
### Describe the project you are working on
RPG

### Describe the problem or limitation you are having in your project
I keep hitting tab to go to the next node in the scene tree when changing node names but it does nothing.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation
In Krita, if you're editing a layer name and hit tab, it changes the layer name and then lets you edit the next node's name. Also shift-tab goes up the layer stack. 

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams
🤷🏼 I dunno, listen for the "tab" key?

### If this enhancement will not be used often, can it be worked around with a few lines of script?
No.

### Is there a reason why this should be core and not an add-on in the asset library?
Editor UI functionality

**Comment by Calinou** [comment]
This probably needs to be added in Tree altogether, so that it works in any tree dialogs in the editor (and project).

**Comment by MaaaxiKing** [comment]
Good idea, but more important would be to switch Scenes instead of Nodes. Maybe that could be achieved by <kbd>Ctrl</kbd>+(<kbd>Shift</kbd>+)<kbd>Tab</kbd>.

**Comment by Calinou** [comment]
> Good idea, but more important would be to switch Scenes instead of Nodes. Maybe that could be achieved by Ctrl+(Shift+)Tab.

This is already supported, but the shortcuts are currently broken: https://github.com/godotengine/godot/pull/47671
